src/uwtools/config/formats/yaml.py

```diff
@@ -145,3 +141,52 @@ def get_format() -> str:
         Returns the config's format name.
         """
         return FORMAT.yaml
+
+    # Private methods
+
+    @classmethod
+    def _add_yaml_representers(cls) -> None:
+        """
+        Add representers to the YAML dumper for custom types.
+        """
+        yaml.add_representer(UWYAMLConvert, UWYAMLConvert.represent)
+        yaml.add_representer(Namelist, cls._represent_namelist)
+        yaml.add_representer(OrderedDict, cls._represent_ordereddict)
+
+    @classmethod
+    def _represent_namelist(cls, dumper: yaml.Dumper, data: Namelist) -> yaml.nodes.MappingNode:
+        """
+        Convert an f90nml Namelist to an OrderedDict, then represent as a YAML mapping.
+
+        :param dumper: The YAML dumper.
+        :param data: The f90nml Namelist to serialize.
+        """
+        namelist_dict = data.todict()
+        return dumper.represent_mapping("tag:yaml.org,2002:map", namelist_dict)
+
+    @classmethod
+    def _represent_ordereddict(
+        cls, dumper: yaml.Dumper, data: OrderedDict
+    ) -> yaml.nodes.MappingNode:
+        """
+        Recursrively convert an OrderedDict to a dict, then represent as a YAML mapping.
+
+        :param dumper: The YAML dumper.
+        :param data: The OrderedDict to serialize.
+        """
+
+        return dumper.represent_mapping("tag:yaml.org,2002:map", from_od(data))
+
+
+def _write_plain_open_ended(self, *args, **kwargs) -> None:
+    """
+    Write YAML without ...
+
+    end-of-stream marker.
+    """
+    self.write_plain_base(*args, **kwargs)
+    self.open_ended = False
+
+
+setattr(yaml.emitter.Emitter, "write_plain_base", yaml.emitter.Emitter.write_plain)
+setattr(yaml.emitter.Emitter, "write_plain", _write_plain_open_ended)
```

Comment:
These two `setattr()` calls and the `_write_plain_open_ended()` function they refer to are responsible for removing the `...` end-of-stream marker from the YAML-scalar output.

Reply:
aha, nice!